server/graphql/vvm/src/mutations/vvm_status_log/update.rs

```diff
@@ -0,0 +1,94 @@
+use async_graphql::*;
+use graphql_core::{
+    standard_graphql_error::{
+        validate_auth,
+        StandardGraphqlError::{BadUserInput, InternalError},
+    },
+    ContextExt,
+};
+use graphql_types::types::IdResponse;
+use repository::vvm_status::vvm_status_log_row::VVMStatusLogRow;
+
+use service::{
+    auth::{Resource, ResourceAccessRequest},
+    vvm::vvm_status_log::update::{
+        UpdateVVMStatusLogError as ServiceError, UpdateVVMStatusLogInput as ServiceInput,
+    },
+};
+
+#[derive(InputObject)]
+pub struct UpdateVVMStatusLogInput {
+    pub id: String,
+    pub status_id: Option<String>,
+    pub comment: Option<String>,
+}
+
+impl UpdateVVMStatusLogInput {
+    pub fn to_domain(self) -> ServiceInput {
+        let UpdateVVMStatusLogInput {
+            id,
+            status_id,
+            comment,
+        } = self;
+
+        ServiceInput {
+            id,
+            status_id,
+            comment,
+        }
+    }
+}
+
+#[derive(Union)]
+pub enum UpdateVVMStatusResponse {
+    Response(IdResponse),
+}
+
+pub fn update_vvm_status_log(
+    ctx: &Context<'_>,
+    store_id: &str,
+    input: UpdateVVMStatusLogInput,
+) -> Result<UpdateVVMStatusResponse> {
+    // TODO: Change this to proper VVM-specific permission
```

Comment:
Is there an issue for this? Is this mentioned in it?

Reply:
Yup, issue is here: https://github.com/msupply-foundation/open-msupply/issues/7692. There's PR up for it, mentioned in the issue

Reply:
We onto it 👯